src/components/Calendar/Filters.tsx

```diff
@@ -35,21 +34,44 @@ const CheckboxAndTitleDiv = styled.div`
     min-width: 300px;
 `;
 
+const SelectAllCheckboxes = styled(Checkbox)``;
```

Comment:
Shall we just use `Checkbox` in the component instead and remove this const if there's no custom styling?

Reply:
Changed